app/views/rooms/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<%= provide(:title, "Show") %>
+<div class="mt-4">
+    <% @rooms.each do |room| %>
+        <div class="col-10">
+            <%= image_tag room.images.first.image, class: "d-block mx-auto img-fluid w-50" %>
+        </div>
+    <% end %>
+</div>
+<div class="ml-4 mt-4">   
+        <h1><%= @room.listing_name %></h1>  
+        <h6><%= @room.address %></h6>
+</div>
+<hr width="1200" align="center">
+<div class="ml-4 mt-4">
+    <h4>About this listings</h4>
+    <h6><%= @room.summery %></h6>
+</div>
+<hr width="1200" align="center">
+<div class="ml-4 mt-4">
+    <h4>Amenities</h4>
+</div>
```

Comment:
please display icons or text that shows if the amenities is there or not
examples below:
![image](https://user-images.githubusercontent.com/73863408/133408968-065b7525-2b62-441d-8705-26e196856420.png)

![image](https://user-images.githubusercontent.com/73863408/133408884-f0ead770-0e93-4487-8527-0e3a000c5265.png)